<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: #292b34</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background:#292b34</string>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background:#292b34</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionUnmute"/>
   <addaction name="actionMute"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #292b34</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAuthor"/>
   <addaction name="actionAuthorName"/>
   <addaction name="actionLikes"/>
   <addaction name="actionLikes_2"/>
   <addaction name="actionComments"/>
   <addaction name="actionComments_2"/>
   <addaction name="actionShare"/>
   <addaction name="actionShare_2"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/refresh.png</normaloff>:/images/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/play.png</normaloff>:/images/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/pause.png</normaloff>:/images/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/next.png</normaloff>:/images/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/previous.png</normaloff>:/images/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionLikes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/like.svg</normaloff>:/images/icons/like.svg</iconset>
   </property>
   <property name="text">
    <string>Likes</string>
   </property>
  </action>
  <action name="actionComments">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/comment.svg</normaloff>:/images/icons/comment.svg</iconset>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/share.svg</normaloff>:/images/icons/share.svg</iconset>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
  </action>
  <action name="actionLikes_2">
   <property name="text">
    <string>Likes</string>
   </property>
  </action>
  <action name="actionComments_2">
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionShare_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/author.png</normaloff>:/images/icons/author.png</iconset>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionAuthorName">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/mute.png</normaloff>:/images/icons/mute.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionUnmute">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/unmute.png</normaloff>:/images/icons/unmute.png</iconset>
   </property>
   <property name="text">
    <string>Unmute</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
